components/Articles.js

```diff
@@ -3,15 +3,25 @@ import { useChannel } from "@ably-labs/react-hooks";
 import ArticlePreview from "./ArticlePreview";
 import styles from "../styles/Home.module.css";
 
-export default function Articles() {
+let clearHistoryState = true;
```

Comment:
Add a comment to this variable to explain why it's here. The variable name is good but I do think this is one of those cases where a comment would help clarify.